apps/minifront/src/components/swap/layout.tsx

```diff
@@ -3,23 +3,26 @@ import { SwapForm } from './swap-form';
 import { UnclaimedSwaps } from './unclaimed-swaps';
 import { AuctionList } from './auction-list';
 import { SwapInfoCard } from './swap-info-card';
+import { LayoutGroup } from 'framer-motion';
 
 export const SwapLayout = () => {
   return (
     <RestrictMaxWidth>
-      <div className='grid w-full grid-std-spacing md:grid-cols-3'>
-        <div className='flex flex-col overflow-hidden grid-std-spacing md:col-span-2'>
-          <SwapForm />
+      <LayoutGroup>
```

Comment:
Added this wrapper so that all animated children get moved in sync. The rest of this file is just indentation changes.